tests/posix/strdupa.c

```diff
@@ -0,0 +1,14 @@
+#ifndef _GNU_SOURCE
+#define _GNU_SOURCE
+#endif
+#include <assert.h>
+#include <string.h>
+
+int main() {
+	char test[19] = "Hello mlibc World!";
```

Comment:
would be good to add an expected fail test case that tries to `free` the `strdupa`d buffer

Reply:
That would be UB, though.

Reply:
yes, hence it's expected to fail